community/kernel/src/test/java/org/neo4j/unsafe/impl/batchimport/input/csv/CsvInputBatchImportIT.java

```diff
@@ -482,10 +504,26 @@ private void buildUpExpectedData(
             for ( int i = 0; i < node.propertyCount(); i++ )
             {
                 final Object expectedValue = node.propertyValue( i );
-                Consumer verify = actualValue ->
+                Consumer verify;
+                if ( expectedValue instanceof TemporalAmount )
                 {
-                    assertEquals( expectedValue, actualValue );
-                };
+                    // Since there is no straightforward comparison for TemporalAmount we add it to a reference
+                    // point in time and compare the result
```

Comment:
I don't understand this. DurationValues implement equals in a sensible way.

Reply:
But `DurationValue` is not a publicly exposed type. I am trying to use only what you would get out of the Core API for verification.